text/0655-enhanced-l1.md

```diff
@@ -44,8 +44,18 @@ This disconnect between L1s and L2s creates challenges for developers trying to
 
 ## Proposed Solution
 
+To resolve these issues in L1s we're proposing enhancing the current L1s by
+1. Expanding the current list of data sources from which we generate L1s.
```

Comment:
Nit: item 1 here is not really "solving any problem". It's the underlying mechanism you are using to implement the solutions described below. Probably not needed.